lineman/app/models/base_model.coffee

```diff
@@ -27,7 +27,10 @@ angular.module('loomioApp').factory 'BaseModel', ->
       @updateFromJSON(data)
 
     clone: ->
-      attrs = @baseSerialize()[@constructor.singular] or {}
+      attrs = _.reduce @constructor.attributeNames, (clone, attr) =>
```

Comment:
Also this use of reduce is fancy schmancy and I'm happy to consider rewriting it to be a little friendlier.